modules/script/script-graaljs-impl/src/main/java/com/enonic/xp/script/impl/executor/ScriptExportsImpl.java

```diff
@@ -49,13 +53,16 @@ public ScriptValue executeMethod( final String name, final Object... args )
             return null;
         }
 
-        try
-        {
-            return method.call( args );
-        }
-        catch ( StackOverflowError e )
+        synchronized ( context )
```

Comment:
not needed sync here, because object creation is wrapped already in doExecuteMain

Reply:
left without changes